draft-irtf-cfrg-vdaf.md

```diff
@@ -3369,19 +3386,20 @@ def eval(self, meas, joint_rand, _num_shares):
     for b in meas:
         out += r * self.GADGETS[0].eval(self.Field, [b])
         r *= joint_rand[0]
-    return out
-~~~
-
-| Parameter        | Value                     |
-|:-----------------|:--------------------------|
-| `GADGETS`        | `[Range2]`                |
-| `GADGET_CALLS`   | `[bits]`                  |
-| `MEAS_LEN`       | `bits`                    |
-| `OUTPUT_LEN`     | `1`                       |
-| `JOINT_RAND_LEN` | `1`                       |
-| `Measurement`    | `int` in `range(2**bits)` |
-| `AggResult`      | `int`                     |
-| `Field`          | `Field128` ({{fields}})   |
+    return [out]
+~~~
+
+| Parameter         | Value                     |
```

Comment:
Will there be a spec for `Sum2` now that we've added a reference implementation?

Reply:
for now `Sum2` is meant to exercise the new feature in tests. It may eventually replace the circuit in `Prio3Sum`, but that's a question for #306.